python/nammu/controller/MenuController.py

```diff
@@ -36,6 +35,9 @@ def saveFile(self):
     def closeFile(self):
         self.controller.closeFile()
     
+    def printFile(self):
```

Comment:
for method in  ['newFile', 'openFile', 'saveFile', 'closeFile']:
       # A lambda for delegation to controller, closes over method.
       def _delegate(self):
                 getattr(self.controller, method)()
      # Set attribute on the MenuController class, delegating.
       MenuController.setattr(method, delegate)